google-cloud-tools-plugin/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
         exclude group: 'com.google.guava', module: 'guava'
     }
     compile files('lib/google-api-services-source.jar')
+//    compile files('lib/libsource-v1-java.jar')
```

Comment:
thats not actually the correct API either. but yeah - this will be updated to use the correct API before merging.